src/apollo/queries.js

```diff
@@ -30,6 +29,19 @@ export const GET_RUN_METADATA = gql`
   }
 `;
 
+/** subscribe to receive new runs */
+export const NEW_RUN_SUBSCRIPTION = gql`
```

Comment:
I might actually put this in its own file. We have `queries.js` and `mutations.js`, so we could also have `subscriptions.js`.